amazon-dynamodb-quickstart/pom.xml

```diff
@@ -73,12 +73,12 @@
         </dependency>
         <dependency>
```

Comment:
These explicit dependencies on testcontainers should not be necessary with devcontainers. I think it is some leftovers.

Reply:
Makes sense, I dropped them, thanks!